pkg/plugin/utils/utils_windows.go

```diff
@@ -0,0 +1,14 @@
+package utils
```

Comment:
no need for  `//go:build windows`  here?

Reply:
Go is supposed to automagically build this file for Windows because it ends with `_windows`.

> The naming convention is described in the documentation for the go/build package. Simply put, if your source file includes the suffix, _$GOOS.go then it will only be built on that platform. All other platforms will behave as if the file is not present. The same applies for _$GOARCH.go. The two can be combined as \_$GOOS\_$GOARCH.go, but not \_$GOARCH\_$GOOS.go.

https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool